test/data/keepalived.conf.tmpl

```diff
@@ -39,7 +39,7 @@ vrrp_instance {{.Cluster.Name}}_INGRESS {
         auth_pass {{.Cluster.Name}}_ingress_vip
     }
     virtual_ipaddress {
-        {{.Cluster.IngressVIP}}/{{.Cluster.VIPNetmask}} label {{.VRRPInterface}}:vip
+        {{.Cluster.IngressVIP}}/{{.Cluster.VIPNetmask}} vip
```

Comment:
Did you mean to drop "label" from this line?

Reply:
No, of course no. Luckily this code here is just a test and not runtime